src/rewriter/rewrite_db_proof_cons.h

```diff
@@ -183,9 +184,29 @@ class RewriteDbProofCons : protected EnvObj
    * @param stepLimit The step limit for this call.
    * @param subgoals The list of proofs introduced when proving eq that
    * are trusted steps.
+   * @param tmode Determines if/when to try THEORY_REWRITE.
+   * @return true if we successfully added a proof of (= a b) to cdp
+   */
+  bool proveStratified(CDProof* cdp,
```

Comment:
Nit (feel free to ignore): rename to `proveEqStratified` and move declaration and definition below `proveEq`.